data/mrs.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+import requests
+
+from data.models import (
+    MergeRequest,
+    Label,
+    IssueNumber,
+    )
+from data.newcomers import active_newcomers
+from data.models import Contributor
+from data.webservices import webservices_url
+from data.convert import convert
+
+
+def fetch_mrs(hoster):
+    """
+    Get mrs opened by newcomers
+
+    :param hoster: a string representing hoster, e.g. 'github'
+    :return: a json of mrs data
+    """
+    logger = logging.getLogger(__name__)
+    if hoster == 'github':
+        IMPORT_URL = webservices_url('mrs/github/all')
+    elif hoster == 'gitlab':
+        IMPORT_URL = webservices_url('mrs/gitlab/all')
+
+    headers = {'Content-Type': 'application/json'}
+    try:
+        response = requests.get(
+            url=IMPORT_URL,
+            headers=headers,
+        )
+        response.raise_for_status()
+    except Exception as e:
+        logger.error(e)
+        return
+    mrs = response.json()
+
+    # Removing mrs which are not opened by newcomers
+    _mrs = []
+    for mr in mrs:
+        if mr['author'] in active_newcomers():
+            _mrs.append(mr)
+    return _mrs
+
+
+def import_mr(hoster, mr):
+    """
+    Import mr data to database
+
+    :param hoster: a string representing hoster
+    :param mr: a dict containing mr's data
+    """
+    logger = logging.getLogger(__name__)
+    number = mr.get('number')
+    assignees = mr.pop('assignees')
+    labels = mr.pop('labels')
+    author = mr.pop('author')
+    repo_id = mr['repo_id']
+    closes_issues = mr.pop('closes_issues')
+    mr['created_at'] = convert(mr['created_at'])
+    mr['updated_at'] = convert(mr['updated_at'])
+    try:
+        c = Contributor.objects.get(login=author)
+        mr['author'] = c
+        mr['hoster'] = hoster
+        m, created = MergeRequest.objects.get_or_create(
+            **mr
```

Comment:
there is no benefit in putting `**` style args on this line, as nothing can be added after this line.  Put it all on one line.